t/01-distributed.t

```diff
@@ -0,0 +1,205 @@
+#!perl
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use Test::Warn;
```

Comment:
Better to use [Test::Warnings](https://metacpan.org/pod/Test::Warnings) - it includes the check for no unexpected warnings as default. Also missing from `cpanfile`.